suites/pacific/rbd/tier-2_rbd_regression.yaml

```diff
@@ -161,3 +161,17 @@ tests:
       module: rbd_snapshot_rename_advanced.py
       name: Test Snapshot Rename with clone operations in progress
       polarion-id: CEPH-9836
+
+  - test:
+      desc: Trying to delete a protected snapshot should fail
+      config:
+        operations:
+          map: true
+          io: true
+          nounmap: true
+      module: rbd_delete_protected_snapshot_krbd.py
+      name: krbd client - Perform Deletion of protected snapshot
```

Comment:
Please start test names with  keyword `Test`  e.g `Test to perform Deletion of protected snapshot`